### PR TITLE
Pull Request: Routing Practical 5 VueJS

### PR DESCRIPTION
**Pull request by Vijay-Thumar**
Live link ---> https://vuejs-practical-5-routing-vijay-thumar.netlify.app/#/

### CODE REVIEW

File: Practical5/src/components/GalleryCard.vue [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  },
+  mounted() {
+    this.fatchCarDetails();
+    let localData = JSON.parse(localStorage.getItem("userAuth"));
+    if(localData == false){
+      this.$router.push("/");
+    }
+  },
+  methods: {
+    async fatchCarDetails() {
```
Comment (k-ekta) [inline Practical5/src/components/GalleryCard.vue]: check spelling here

File: Practical5/src/components/CarForm.vue [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+                @click="closeClickHandler"
+              >
+                Close
+              </button>
+            </Form>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div v-else></div>
```
Comment (k-ekta) [inline Practical5/src/components/CarForm.vue]: remove v-else if not needed

File: Practical5/src/components/AddNewCar.vue [hunk 3]
```diff
@@ -0,0 +1,209 @@
+0<template>
```
Comment (k-ekta) [inline Practical5/src/components/AddNewCar.vue]: remove 0 here

File: Practical3/src/components/AddCarForm.vue [hunk 4]
```diff
@@ -0,0 +1,139 @@
+<template>
+  <div>
```
Comment (k-ekta) [inline Practical3/src/components/AddCarForm.vue]: make one form for add and edit

**Comment by k-ekta** [comment]
for register form make form fields vertically

**Comment by k-ekta** [comment]
add toast message when error is shown shown

File: Practical5/src/data/jsonData.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+[
+
```
Comment (k-ekta) [inline Practical5/src/data/jsonData.json]: no data found in this file

File: Practical5/src/views/MyCar.vue [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+  data() {
+    return {
+      id: this.$route.params.id,
+      myCar: object,
+      loading: true,
+      apiError: false,
+    };
+  },
+  mounted() {
+    this.fatchMyCar();
```
Comment (k-ekta) [inline Practical5/src/views/MyCar.vue]: correct spelling here

File: Practical5/src/views/MyCar.vue [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+</template>
+
+<script>
+import axios from "axios";
+import { object } from "yup";
+export default {
+  name: "MyCar",
+  data() {
+    return {
+      id: this.$route.params.id,
```
Comment (k-ekta) [inline Practical5/src/views/MyCar.vue:42]: Add branching here. What if initialy id does not exist it will break from here

Comment (Vijay-Thumar) [reply Practical5/src/views/MyCar.vue:42]: This page aka component won't open if id is not available.

**Pull request merged**